mmv1/products/sql/terraform.yaml

```diff
@@ -83,6 +85,9 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       onPremisesConfiguration.port: !ruby/object:Overrides::Terraform::PropertyOverride
         validation: !ruby/object:Provider::Terraform::Validation
           function: 'validation.IntBetween(0, 65535)'
+      onPremisesConfiguration.password: !ruby/object:Overrides::Terraform::PropertyOverride
+        sensitive: true
+        custom_flatten: templates/terraform/custom_flatten/source_representation_instance_configuration_password.go.erb
```

Comment:
Can this be replaced by ignore_read on the field instead?

Reply:
I tried with ignore_read on this field and it didn't work as expected. 
Since this field is nested, I was told that ignore_read doesn't work on nested field, and instead a custom_flatten should be used here.

Reply:
Ah, ok sounds good